README.md

```diff
@@ -62,6 +62,206 @@ Note: Please use [google-cloud-bigtable][google-cloud-bigtable] to access Bigtab
+    <version>0.24.0</version>
+    <scope>runtime</scope>
+</dependency>
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
+    <version>0.24.0</version>
+    <exclusions>
+        <exclusion>
+            <groupId>io.grpc</groupId>
+            <artifactId>*</artifactId>
+        </exclusion>
+        <exclusion>
+            <groupId>com.google.auth</groupId>
+            <artifactId>*</artifactId>
+        </exclusion>
+    </exclusions>
+</dependency>
+```
+
+If you _are_ using the shaded Bigtable HBase Client artifact, then the OpenCensus dependencies are embedded in the
+shaded artifact; i.e. nothing additional for you to do.
+
+```xml
+<!-- OpenCensus dependencies -->
+<dependency>
+    <groupId>com.google.cloud.bigtable</groupId>
+    <artifactId>bigtable-hbase-1.x-shaded</artifactId>
+    <version>1.14.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+##### Java Example
+```java
+// For the non-shaded client, remove the package prefix "com.google.bigtable.repackaged."
+import com.google.bigtable.repackaged.io.opencensus.contrib.grpc.metrics.RpcViews;
+import com.google.bigtable.repackaged.io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
+import com.google.bigtable.repackaged.io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+
+import java.io.IOException;
+
+public class OpenCensusExample {
+    String projectId = "your-project-id";
+
+    void setupStatsExport() throws Exception {
+        // Option 1: Automatic Configuration (from GCP Resources only):
+        // If you are running from a GCP Resource (e.g. a GCE VM), the Stackdriver metrics are automatically
+        // configured to upload to your resource.
+        // For examples of monitored resources, see here: https://cloud.google.com/monitoring/api/resources
+        StackdriverStatsExporter.createAndRegister();
+
+        // Then register your gRPC views in OpenCensus.
+        RpcViews.registerClientGrpcViews();
+
+
+        // Option 2: Manual Configuration
+        // If you are not running from a GCP Resource (e.g. if you are running on-prem), then you should
+        // configure the monitored resource yourself.
+        // Use the code snippet below as a starting example.
+        // For examples of monitored resources, see here: https://cloud.google.com/monitoring/api/resources
+        StackdriverStatsExporter.createAndRegister(
+                StackdriverStatsConfiguration.builder()
+                        .setProjectId(projectId)
+                        // This example uses generic_node as the MonitoredResource, with your host name as the node ID.
+                        .setMonitoredResource(MonitoredResource.newBuilder()
+                                .setType("generic_node")
+                                .putLabels("project_id", projectId)
+                                .putLabels("location", "us-west1-b")
```

Comment:
Should we also add a comment here, something like "The region in which the service is running, in this case us-west1-b"?

Reply:
Done